hera_cal/vis_clean.py

```diff
@@ -486,6 +590,24 @@ def fourier_filter(self, filter_centers, filter_half_widths, mode,
         verbose : Lots of outputs.
         overwrite : bool, if True, overwrite output modules with the same name
                     if they already exist.
+        skip_flagged_edge_freqs : bool, optional
+            if true, do not filter over flagged edge frequencies (filter over sub-region)
+            defualt is False
+        skip_flagged_edge_times : bool, optional
+            if true, do not filter over flagged edge times (filter over sub-region)
+            defualt is False
+        skip_gaps_larger_then_filter_period : bool, optional
+            if true, skip integrations or channels with gaps that are larger then the period of
+            of the finest scale mode used for interpolation.
+        flag_filled : bool, optional
+            if true, set filter flags equal to the original flags (do not unflag interpolated channels)
+            This is useful for cross-talk filtering where the cross-talk modes will not completely interpolate
+            over the channel gaps since there are substantial contributions to the foreground power from fringe-rates
+            that are not being modeled as cross-talk. In this case, we may want a file with the modelled cross-talk included but
+            not used to in-paint flagged integrations.
+        clean_flags_in_resid_flags : bool, optional
+            if true, include clean flags in residual flags that will be written out in res_outfilename.
+            default is False.
```

Comment:
I know this is not technically part of this PR, but I think it is good to have an enumeration of all of the filter_kwargs somewhere in hera_cal, which you can then reference from higher level wrappers like vis_clean, or delay_clean etc. In other words, we shouldn't reference all the way to `uvtools`, and should probably put all of the kwargs and their docstrings at least once at the lowest level hera_cal method, which I think is here. I guess part of my thought here is, some of the enumerated kwargs are indeed filter_kwargs--what made you decide which to enumerate and which to package up in the **filter_kwargs dict? If we can't think of a good reason why, perhaps its better to enumerate them here. But I do acknowlegde it can get tedious every time, which is why I think its probably fine to do this in higher level wrappers within the hera_cal repo

Reply:
I think the `filter_kwargs` here are basically set by `uvtools.dspec.fourier_filter` and the choice of which args to make kwargs there was set by options that were different depending on whether the filtering mode chosen (`clean` vs. `dayenu` vs. `dpss_leastsq` etc...). I think these have been enumerated here.

Maybe you are saying that we should put more args into kwargs at higher levels?

Reply:
no, I'm just saying that we might want to enumerate all of the possible kwargs at some point here, but on second thought that's tough b/c it will change as uvtools is developed. as long as its clear that the additional kwargs are specific to additional kwargs in `uvtools.dspec.fourier_filter` I'm fine w/ it. In other words, ppl know where to look if they are wondering what else might go in this open-ended kwarg slot.

Reply:
I've added a note that these kwargs are enumerated in `uvtools.dspec.fourier_filter`.